oauth2-server/src/main/java/com/clouway/oauth2/codechallenge/AuthorizationCodeVerifier.java

```diff
@@ -0,0 +1,48 @@
+package com.clouway.oauth2.codechallenge;
+
+import com.google.common.base.Strings;
+import com.google.common.io.BaseEncoding;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+/**
+ * @author Vasil Mitov <vasil.mitov@clouway.com>
+ */
+public class AuthorizationCodeVerifier implements CodeVerifier {
+  @Override
+  public boolean verify(CodeChallenge codeChallenge, String codeVerifier) {
```

Comment:
s/codeVerifier/requestedCode/g

Reply:
I haven't seen these aliases in a while and i can't really find their meanings. Did this mean replace ?

Reply:
I don't think requestedCode sounds good. It is providedCode as provided from the client that tries to authenticate. Maby providedCodeVerifier would be better ?